R/run_app.R

```diff
@@ -1,29 +1,41 @@
 #' Run the Shiny Application
 #'
-#' @param ... arguments to pass to golem_opts.
-#' See `?golem::get_golem_options` for more details.
+#' @param app_ver a "global" variable that is passed to several modules &
+#'   reports which details the installed package version when not specified.
+#'   It can be overwritten to include a specific version name as a text string.
+#' @param login_note a text string to display underneath the auth screen's login
+#'   button, provided to guide users
+#' @param ... arguments to pass to golem_opts. See `?golem::get_golem_options`
+#'   for more details.
 #' @inheritParams shiny::shinyApp
 #'
 #' @export
 #' @importFrom shiny shinyApp
 #' @importFrom golem with_golem_options
 run_app <- function(
-  onStart = initialize_raa,
+  onStart = initialize_raa(),
```

Comment:
The parentheses `()` should not be present. This caused my app to error out on a fresh initialization.

Reply:
Oh sorry. I actually was testing out adding a argument to this function called "ver" to handle the app_version stuff. It actually worked well, but I felt it was weird calling `run_app()` like this:

```
run_app(onStart = initialize_raa(ver = "my version"))
```
so I abandoned it but forgot to remove the `( )`.